### PR TITLE
Filter tests based on environment 

### PR DESCRIPTION
**Pull request by shahthepro**
If you made a contract change, make sure to complete the checklist below before merging it in master.

Refer to our [documentation](https://github.com/OriginProtocol/security) for more details about contract security best practices.

Contract change checklist:
  - [ ] Code reviewed by 2 reviewers. 
  - [ ] Copy & paste code review [security checklist](https://github.com/OriginProtocol/security/blob/master/templates/Contract-Code-Review-Example.md) below this checklist.
  - [ ] Unit tests pass
  - [ ] Slither tests pass with no warning
  - [ ] Echidna tests pass if PR includes changes to OUSD contract (not automated, run manually on local)

### CODE REVIEW

**Review by naddison36 — DISMISSED** [review]
nice solution. Well done

File: contracts/test/_global-hooks.js [hunk 1]
```diff
@@ -1,34 +1,56 @@
 const mocha = require("mocha");
 
+const { isForkTest } = require("./helpers");
+
 const _chunkId = Number(process.env.CHUNK_ID);
 const _maxChunks = Number(process.env.MAX_CHUNKS);
```
Comment (naddison36) [inline contracts/test/_global-hooks.js:6]: It'd be nice to have these env variables documented in a README

**Review by sparrowDom — DISMISSED** [review]
LGTM

File: contracts/test/_global-hooks.js (hunk 1 above)
Comment (shahthepro) [reply contracts/test/_global-hooks.js:6]: Have added them now

**Pull request merged**